src/components/DataTable/DataTable.component.tsx

```diff
@@ -25,7 +26,7 @@ export const DataTable = <Data extends BaseRowData = BaseRowData>({
   tableAction,
   containerSx,
 }: DataTableProps<Data>) => {
-  const isUpdate = isLoading && data.length;
+  const isUpdate = isLoading && data.length > 0;
```

Comment:
What's better with this approach?

Reply:
we get returned '0' if isLoading true, so types were number | boolean | undefined or null, don't remember